src/docs/markdown/caddyfile/concepts.md

```diff
@@ -145,7 +145,7 @@ Inside quoted tokens, all other characters are treated literally, including spac
 
 ```caddy-d
 directive "first line
-second line"
+	second line"
```

Comment:
Is this change just to illustrate that tabs are taken literally inside quotes?

Reply:
Yeah, to show whitespace is preserved/allowed